backend/src/apiserver/model/job.go

```diff
@@ -89,10 +89,15 @@ func (j *Job) DefaultSortField() string {
 }
 
 var jobAPIToModelFieldMap = map[string]string{
-	"id":         "UUID",
-	"name":       "DisplayName",
-	"created_at": "CreatedAtInSec",
-	"package_id": "PipelineId",
+	"id":            "UUID",
+	"name":          "DisplayName",
+	"created_at":    "CreatedAtInSec",
+	"package_id":    "PipelineId",
+	"finished_at":   "FinishedAtInSec",
```

Comment:
Since I am making changes to `model/run.go`, I think I can patch the similar issue with jobs. But you are right, we don't have `FinishedAtInSec ` in `Job` type. I will remove it.